src/mame/nec/pc88va.cpp

```diff
@@ -256,7 +256,7 @@ void pc88va_state::sys_port5_w(u8 data)
 
 u8 pc88va_state::sys_port5_r()
 {
-	return m_rstmd | 8;
+	return (u8)m_rstmd | 8;
```

Comment:
Better to use `(m_rstmd ? 1 : 0) | 8` to explicitly map `bool` to integer values.